test/e2e/airgap.go

```diff
@@ -0,0 +1,20 @@
+package e2e
+
+import (
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+// Run airgap deployment but allow bootstrap cluster to access local peers.
+func runVSphereAirgapConfigFlow(test *framework.ClusterE2ETest, localCIDRs string) {
+	test.GenerateClusterConfig()
+	test.DownloadArtifacts()
+	test.ImportImages()
```

Comment:
Based on offline conversation I think we eventually want this changed to use `import images` instead of `import-images`, right? Happy to merge as is, just flagging.

Reply:
> Just a few small nits/notes.
> 
> Discussed with @d8660091 adding a step to verify that the environment is indeed airgapped, for instance by making a http call against something on the public internet with a short timeout. He's going to do it in a fast follow PR.
> 
> /lgtm

The task is created: https://github.com/aws/eks-anywhere/issues/4980

Reply:
Yes, we will deprecate import-images perhaps all at once, which is out of the scope of this PR